manifests/params.pp

```diff
@@ -33,6 +33,15 @@
         $package_name = 'iptables-persistent'
       }
     }
+    'Ubuntu' : {
```

Comment:
The cleanest way to do this is to move this logic into the Debian case and make it look similar to the RedHat case (a nested case for operatingsystem).